### DISCUSSION TITLE
原点復帰コマンドについて

### DISCUSSION THREAD

**Discussion by bbm-soka** [ponoor/step-series-support#29] [Q&A] [answered]
モータースライダーでSTEP400デバイスを使用しています。

"homing"コマンドを入力すると、HOMEセンサの位置まで移動し停止するところまではうまく動作しています。その後、スライダーがHOMEセンサが反応しなくなる位置まで移動するはずですが、うまく動作していないようです。

"homingStatus"の状態としては、0→1→2→4という遷移になっていました。

おそらくですが、使用しているモータースライダーの精度がよくなく、"releaseSw"コマンドの初期設定のminimum speedだとスピードが小さく動いていないのではと考えています。

そこで、"releaseSw"コマンドで使用されるminimum speedはどのコマンドで変更できますでしょうか？

よろしくお願いします。

**Comment by bbm-soka** [comment, marked as answer]
過去のフォーラムを見逃していました。。

同じ質問がありましたので試してみます！
https://github.com/ponoor/step-series-support/discussions/4

ちなみに、ファームウェアを最近アップデートしたばかりですが、対応されてますでしょうか？

**Reply by kanta** [reply]
v1.0.3以降のファームウェアでしたら /setMinSpeed コマンドに対応しています。
https://ponoor.com/docs/step-series/osc-command-reference/speed-profile/#setminspeed_intmotorid_floatminspeed

ここはとても分かりにくい状態になっていますね、、、近いうちにドキュメントを改善したいと思います。
